### PR TITLE
fix: like hover color, button size and skeleton UI for card

### PR DESCRIPTION
**Pull request by revengemiroz**
# Description

- Fix : Skeloton of card not matching the ui
- Fix : On hover links on navbar theres no hover color
- Fix :  Remove unused codes
## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

-  My code follows the style guidelines of this project
-  I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Showcase

Please attach a picture or a video showing what this change does.

### CODE REVIEW

File: apps/web/src/components/Overlay/Overlay.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+const Overlay = ({ children, color }: OverlayProps) => {
+  return (
+    <div className='relative'>
+      {children}
+      <div
+        className={`absolute inset-0 bg-${color} opacity-75`}
+        style={{ zIndex: 10 }}
+      ></div>
```
Comment (ivanms1) [inline apps/web/src/components/Overlay/Overlay.tsx]:
```suggestion
     />
```

File: packages/ui/Button/index.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ interface Button extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   size?: 'small' | 'medium' | 'large';
   url?: string;
   noAnimation?: boolean;
+  bordeRadius?: string;
```
Comment (ivanms1) [inline packages/ui/Button/index.tsx]: remove this property please, just use the className

File: packages/ui/Button/index.tsx [hunk 3]
```diff
@@ -37,7 +39,7 @@ export const Button = React.forwardRef<HTMLButtonElement, Button>(
       <button
         ref={ref}
         className={classNames(
-          'rounded-[20px] px-12 font-semibold transition-all disabled:cursor-not-allowed disabled:opacity-50',
+          `rounded-[${bordeRadius}] px-12 font-semibold disabled:cursor-not-allowed disabled:opacity-50`,
```
Comment (ivanms1) [inline packages/ui/Button/index.tsx]: revert these changes please

File: apps/web/src/pages/Home/Home.tsx [hunk 4]
```diff
@@ -155,21 +153,16 @@ function Home() {
             </p>
           </div>
           <Link href='/top-projects' passHref>
-            <Button className='max-lg:hidden'>{t('see-all-projects')}</Button>
+            <Button bordeRadius='10px' className='max-lg:hidden'>
```
Comment (ivanms1) [inline apps/web/src/pages/Home/Home.tsx]: add the border radius in the className

File: apps/web/src/pages/Home/Home.tsx [hunk 5]
```diff
@@ -52,7 +52,9 @@ function Home() {
           className='max-lg:mb-10 max-lg:w-full'
           passHref
         >
-          <Button className='max-lg:w-full'>{t('common:add-project')}</Button>
+          <Button bordeRadius='10px' className='max-lg:w-full'>
```
Comment (ivanms1) [inline apps/web/src/pages/Home/Home.tsx]: add radius in className

File: apps/web/src/pages/Home/Home.tsx [hunk 6]
```diff
@@ -129,20 +131,16 @@ function Home() {
             </p>
           </div>
           <Link href='/top-creators' passHref>
-            <Button className='max-lg:hidden'>{t('view-rankings')}</Button>
+            <Button bordeRadius='10px' className='max-lg:hidden'>
```
Comment (ivanms1) [inline apps/web/src/pages/Home/Home.tsx]: add radius in className

**Review by ivanms1 — CHANGES_REQUESTED** [review]
rename PR to a more descriptive name

File: apps/web/src/components/Overlay/Overlay.tsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+interface OverlayProps {
+  children: JSX.Element;
+  color: string;
+}
+
+const Overlay = ({ children, color }: OverlayProps) => {
```
Comment (ivanms1) [inline apps/web/src/components/Overlay/Overlay.tsx]: where is this being used?

File: apps/web/src/components/Overlay/Overlay.tsx (hunk 1 above)
Comment (revengemiroz) [reply apps/web/src/components/Overlay/Overlay.tsx]: removed

File: packages/ui/Button/index.tsx (hunk 2 above)
Comment (revengemiroz) [reply packages/ui/Button/index.tsx]: removed

File: packages/ui/Button/index.tsx (hunk 3 above)
Comment (revengemiroz) [reply packages/ui/Button/index.tsx]: done

File: apps/web/src/pages/Home/Home.tsx (hunk 5 above)
Comment (revengemiroz) [reply apps/web/src/pages/Home/Home.tsx]: done

**Review by ivanms1 — APPROVED** [review]
